roles.sql

```diff
@@ -3,6 +3,7 @@ BEGIN;
 
 CREATE ROLE caa_redirect WITH PASSWORD 'caa_redirect';
 CREATE ROLE critiquebrainz WITH PASSWORD 'critiquebrainz';
+CREATE ROLE listenbrainz WITH PASSWORD 'listenbrainz';
```

Comment:
I think it should be `WITH LOGIN PASSWORD` for the app to work (others are probably wrong too).